WordPress/src/main/java/org/wordpress/android/ui/qrcodeauth/compose/components/SecondaryButton.kt

```diff
@@ -0,0 +1,37 @@
+package org.wordpress.android.ui.qrcodeauth.compose.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.wordpress.android.ui.compose.unit.Margin
+
+@Composable
+fun SecondaryButton(
```

Comment:
You should add a `Modifier` parameter to all of your Composables (source to [guidelines](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#elements-accept-and-respect-a-modifier-parameter))

Reply:
Done in https://github.com/wordpress-mobile/WordPress-Android/pull/16934/commits/acb3aa80a2dd1380a4a07ab5b6ee1033b068bdbd, applying the suggestions from the [guidelines](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#elements-accept-and-respect-a-modifier-parameter) 🙇

Reply:
> I still went with setting the default modifiers inside the component code

Afaict, I think this is ok, since you are adding the additional modifiers at the end of the passed modifier, and not at the beginning ([source](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-api-guidelines.md#elements-accept-and-respect-a-modifier-parameter)).